src/platform/plugins/shared/saved_search/server/content_management/schema/cm_services.ts

```diff
@@ -12,11 +12,14 @@ import type {
   Version,
 } from '@kbn/object-versioning';
 
-// We export the versionned service definition from this file and not the barrel to avoid adding
+// We export the versioned service definition from this file and not the barrel to avoid adding
 // the schemas in the "public" js bundle
 
+import { SAVED_SEARCH_CM_VERSION_1, SAVED_SEARCH_CM_VERSION_2 } from '../../../common/constants';
 import { serviceDefinition as v1 } from './v1/cm_services';
+import { serviceDefinition as v2 } from './v2/cm_services';
 
 export const cmServicesDefinition: { [version: Version]: ServicesDefinition } = {
-  1: v1,
+  [SAVED_SEARCH_CM_VERSION_1]: v1,
+  [SAVED_SEARCH_CM_VERSION_2]: v2,
 };
```

Comment:
Re this question:
>Backwards compatibility - should the v1 content managment APIs continue to be functional? (It doesn't appear they are for things like dashboards that have multiple CM versions)

My understanding from @ThomThomson was that we currently don't need to keep the old one around once we've introduced a new version. Maybe he could confirm if I understood that correctly, and if so, should we just delete v1 entirely?